arbeitszeit_web/www/presenters/remove_worker_from_company_presenter.py

```diff
@@ -0,0 +1,47 @@
+from dataclasses import dataclass
+
+from arbeitszeit.use_cases.remove_worker_from_company import Response as UseCaseResponse
+from arbeitszeit_web.notification import Notifier
+from arbeitszeit_web.translator import Translator
+from arbeitszeit_web.url_index import UrlIndex
+from arbeitszeit_web.www.response import Redirect
+
+
+@dataclass
+class ErrorCode:
+    code: int
+
+
+ViewModel = Redirect | ErrorCode
+
+
+@dataclass
+class RemoveWorkerFromCompanyPresenter:
+    notifier: Notifier
+    translator: Translator
+    url_index: UrlIndex
+
+    def present(self, use_case_response: UseCaseResponse) -> ViewModel:
+        if not use_case_response.is_rejected:
+            self.notifier.display_info(
+                self.translator.gettext("Worker successfully removed from company.")
```

Comment:
Let's see — the UseCaseResponse object here does not have identifying information about the worker, does it?  If it does, it would be good for the notifier to display it here as part of the string.

Reply:
No, it hasn't... Would complicate the response logic slightly. Let me know if you would like to have it implemented, than I'll do it.

Reply:
I implemented the rest of your requests, thanks!

Reply:
> No, it hasn't... Would complicate the response logic slightly. Let me know if you would like to have it implemented, than I'll do it.

No, that's OK — thank you!  Do you have time to write it up as a separate issue?  Somebody could go back later and fix it.  I think it would be much better if both the company removing the worker and the worker being removed got confirmations of that action, where the company would know exactly who was removed.  Since removal seems to me pretty serious — and permanent — the system should help the company to avoid mistakes.